{{cookiecutter.project_slug}}/.idea/workspace.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DjangoConsoleOptions" custom-start-script="import sys; print('Python %s on %s' % (sys.version, sys.platform))&#10;import django; print('Django %s' % django.get_version())&#10;import os&#10;os.environ.setdefault(&quot;DATABASE_URL&quot;,&quot;postgres://{}:{}@{}:{}/{}&quot;.format(os.environ['POSTGRES_USER'], os.environ['POSTGRES_PASSWORD'], os.environ['POSTGRES_HOST'], os.environ['POSTGRES_PORT'], os.environ['POSTGRES_DB']))&#10;{%- if cookiecutter.use_celery == 'y' %}os.environ.setdefault(&quot;CELERY_BROKER_URL&quot;, os.environ['REDIS_URL'])&#10;{% endif %}sys.path.extend([WORKING_DIR_AND_PYTHON_PATHS])&#10;if 'setup' in dir(django): django.setup()&#10;import django_manage_shell; django_manage_shell.run(PROJECT_ROOT)" module-name="{{ cookiecutter.project_slug }}" is-module-sdk="true">
+    <option name="myCustomStartScript" value="import sys; print('Python %s on %s' % (sys.version, sys.platform))&#10;import django; print('Django %s' % django.get_version())&#10;import os&#10;os.environ.setdefault(&quot;DATABASE_URL&quot;,&quot;postgres://{}:{}@{}:{}/{}&quot;.format(os.environ['POSTGRES_USER'], os.environ['POSTGRES_PASSWORD'], os.environ['POSTGRES_HOST'], os.environ['POSTGRES_PORT'], os.environ['POSTGRES_DB']))&#10;os.environ.setdefault(&quot;CELERY_BROKER_URL&quot;, os.environ['REDIS_URL'])&#10;sys.path.extend([WORKING_DIR_AND_PYTHON_PATHS])&#10;if 'setup' in dir(django): django.setup()&#10;import django_manage_shell; django_manage_shell.run(PROJECT_ROOT)" />
```

Comment:
Strange, the if statement evaluates to true when not using Celery. 
```
{%- if cookiecutter.use_celery == 'y' %}os.environ.setdefault(&quot;CELERY_BROKER_URL&quot;, os.environ['REDIS_URL'])&#10;{% endif %}
```

Reply:
It's in the `<component name="DjangoConsoleOptions"` line, but there is no condition in the `<option name="myCustomStartScript"` line... `os.environ.setdefault(&quot;CELERY_BROKER_URL&quot;, os.environ['REDIS_URL'])` is there unconditionally.

Given how hard it is to spot it, I'm wonding if adding Jinja templating in this file is a good idea 🤔 Since there are 2 variants only, maybe it would be more maintainable if we had 2 files: `workspace-nocelery.xml` and `workspace-with-celery.xml` and use the post generation hook to rename the one we want to keep and remove the other?

Reply:
Ah, fml. Thanks for helping me out!

The myCustomStartScript isn't even necessary (didn't notice it until you mentioned it).  

I will wrap the `<component />` tag in an if-else so we do not have multiple files to maintain.